tools/flang1/flang1exe/main.c

```diff
@@ -94,6 +94,7 @@ static const char *feature = "pgfortran";
 static const char *feature = "pgfortran";
 #endif
 
+bool g_list_macros = FALSE;
```

Comment:
is this variable read anywhere? I can see it written below, but never actually read.

Reply:
ok, I'm seeing it now, this is the variable shared across those files. it is ok to define it and initialize it here, but it's declaration (with `extern`) should also appear in `main.h`.